book/frontend/tutorial/step_7.md

```diff
@@ -1,42 +1,104 @@
 # Step 7
 
-At buildo, we are not TDD integralists but we believe adding new unit tests should be as painless as possible. This is why the boilerplate includes all you need to get started. This is all the code you need to add in ` test/tests/components/Hello.js` to create a couple of runnable tests for our UI component.
+In the previous step we included a components from our [react-components](https://github.com/buildo/react-components/)by following the [guidelines](../guidelines/5.buildo-react-components.md) that we defined in this book. However, there was no customization involved.
 
+In this step we will improve the UI of our beloved `Hello` component using the b-r-c [Panel](https://github.com/buildo/react-components/tree/master/src/Panel), customized specifically for this project.
+
+## Creating the Basic Panel component
+
+Let's perform the same steps that we did previously for the loading spinner, but this time adding some customization on top of the standard b-r-c component:
+
+1. Create a `src/app/components/Basic/Panel/Panel.js` file as follows:
 ```js
-import Hello from 'components/Hello/Hello.js';
-import expect from 'expect';
+import React from 'react';
+import BRCPanel from 'buildo-react-components/lib/Panel/Panel';
+import skinnable from 'react-skinnable';
 
-const baseProps = {
-  toggle: () => {},
-  user: 'user',
-  onRefreshClick: () => {}
-};
+import './panel.scss';
 
-describe('Hello', () => {
+@skinnable()
+export default class Panel extends React.PureComponent {
 
-  it('should greet in an informal way when formal=false', () => {
-    const props = { ...baseProps, formal: false };
-    const instance = new Hello(props); 
-    const { greeting } = instance.getLocals(props);
-    expect(greeting).toBe('Hello');
+  template({ type = 'docked-top', ...props }) {
+    return <BRCPanel type={type} {...props} />;
+  }
 
-  });
+}
+```
+As you can see, we already customized the `BRCPanel` by saying that we want the `Basic` `Panel` of this project to always be `docked-top`.
 
-  it('should greet in a formal way when formal=true', () => {
-    const props = { ...baseProps, formal: true };
-    const instance = new Hello(props); 
-    const { greeting } = instance.getLocals(props);
-    expect(greeting.includes('Good')).toBe(true);
+2. Export it through an `index.js` file in the same folder:
+```js
+export default from './Panel';
+```
 
-  });
+3. Import it's style in the `panel.scss` file and customize it using the SASS interface:
+```css
+@import '~theme/variables.scss';
 
-});
+$content-background: $cloud;
+
+@import '~buildo-react-components/src/Panel/panel.scss';
+
+.panel {
+  border-radius: 10px;
+  box-shadow: 0 2px 0 $darkGrey;
+
+  .panel-content {
+    border-radius: 10px;
+  }
+}
 ```
+We just overridden its default background color and customized its border.
```

Comment:
I do it for the `$content-background: $cloud;`!
 
Shadow and border-radius are not defined in the original panel.scss I think, so for these I just used normal css style